python/GafferUI/PlugValueWidget.py

```diff
@@ -453,6 +454,17 @@ def __plugFlagsChanged( self, plug ) :
 		if plug.isSame( self.__plug ) :
 			self._updateFromPlug()
 
+	def __plugMetadataChanged( self, nodeTypeId, plugPath, key, plug ) :
+
+		if self.__plug is None :
+			return
+
+		if (
+			Gaffer.affectedByChange( self.__plug, nodeTypeId, plugPath, plug ) or
```

Comment:
> I take it you'd argue "lets introduce one more without the namespace rather than making more big sweeping changes in this pr"?

I would.

> What about the counter "lets merge this as is but immediately put up another PR that changes them all, and have those go out in the same major version release"?

I think I'd probably buy that.

> Actually, I just looked at all the Algo files and I find myself in a pickle... I definitely prefer Gaffer.StringAlgo.hasWildcards but GafferScene.SceneAlgo.parallelTraverse feels a bit redundant... maybe the rule could be "each module can have some non-namespaced functions if they reflect the primary purpose of the module"? Note sure if that makes sense in all cases though...

If we're gonna do this, let's keep it simple - all Algo in a namespace. Better that than always trying to remember which functions you thought warranted special treatment in the past.